gravitee-apim-cypress/cypress/fixtures/json/updateApi_mockPolicy_example.json

```diff
@@ -0,0 +1,74 @@
+{
+  "version": "5.8.13",
+  "description": "Carbonite web goalkeeper gloves are ergonomically designed to give easy fit",
+  "proxy": {
+    "virtual_hosts": [
+      {
+        "path": "/Wooden-24223b8d-99fd-4912-a4b9-d7c70143e8cd-1639991125"
+      }
+    ],
+    "strip_context_path": false,
+    "preserve_host": false,
+    "groups": [
+      {
+        "name": "default-group",
+        "endpoints": [
+          {
+            "backup": false,
+            "name": "default",
+            "weight": 1,
+            "type": "http",
+            "target": "https://api.gravitee.io/echo"
+          }
+        ],
+        "load_balancing": {
+          "type": "ROUND_ROBIN"
+        },
+        "http": {
+          "connectTimeout": 5000,
+          "idleTimeout": 60000,
+          "keepAlive": true,
+          "readTimeout": 10000,
+          "pipelining": false,
+          "maxConcurrentConnections": 100,
+          "useCompression": true,
+          "followRedirects": false
+        }
+      }
+    ]
+  },
+  "paths": {
+    "/": [
+      {
+        "methods": ["GET"],
+        "description": "Description of the Mock Gravitee Policy",
+        "enabled": true,
+        "mock": {
+          "headers": [
+            {
+              "name": "test-value",
+              "value": "value123"
+            }
+          ],
+          "status": "200",
+          "content": "This is a mocked response"
+        }
+      }
+    ]
+  },
+  "visibility": "PRIVATE",
+  "name": "Awesome Frozen Shirt",
+  "services": {},
+  "properties": [],
+  "tags": [],
+  "picture_url": "http://localhost:8083/management/organizations/DEFAULT/environments/DEFAULT/apis/5f697c6b-635a-440e-a97c-6b635a340ed9/picture?hash=1641283944514",
```

Comment:
forbidden ! 
not sure you will always run your test with localhost:8083 ;)

Reply:
I get your point. Ok, I'll just remove all the parts that not needed, including that URL.